owlsim-core/src/test/java/org/monarchinitiative/owlsim/io/OntologySourceDataTest.java

```diff
@@ -18,12 +20,37 @@ public void testItAll() {
         curies.put("MP", "http://purl.obolibrary.org/obo/MP_");
         curies.put("NCBITaxon", "http://purl.obolibrary.org/obo/NCBITaxon_");
 
+        Map<String, List<String>> individuals = new LinkedHashMap<>();
```

Comment:
We should now have a thought about how to add additional information to an annotation. E.g. a weight or something. 

Is it already possible to add negation?